### PR TITLE
Add careers page (In progress)

### PR DESCRIPTION
**Pull request by taniaprocha**

### CODE REVIEW

**Comment by taniaprocha** [comment]
@filipebfernandes I created this PR to have the progress of the careers page. In the apprenticeship, we added Hero and Happiness sections. We'll merge to master only when all the page is ready. Can you review the code of these 2 sections, after I'll ask you section by section.
Thanks

**Comment by filipebfernandes** [comment]
Yes, sure.

File: src/sections/CareersHappiness/index.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  @include Breakpoint-desktopOnly {
+    margin-right: $column-spacing;
+  }
+  @include Breakpoint-tabletOnly {
+    width: 60%;
+    margin-right: $column;
+  }
+
+  .image {
+    margin-left: calc(-1 * $column-spacing);
```
Comment (filipebfernandes) [inline src/sections/CareersHappiness/index.css:34]: not 100% sure but I think we don't need calc for negative spacing

File: src/sections/CareersHero/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+import Section from "root/components/Section";
+import Navbar from "root/components/Navbar";
+import Typography from "root/components/Typography";
+import Wave from "../../assets/images/wave-careers-hero.inline.svg";
```
Comment (filipebfernandes) [inline src/sections/CareersHero/index.js:6]: can't import from root?

File: src/sections/CareersHappiness/index.css (hunk 1 above)
Comment (taniaprocha) [reply src/sections/CareersHappiness/index.css:34]: postcss return an error with `-$column-spacing`, but I don't need calc to `-$gutter` as I used to use.

File: src/sections/CareersHero/index.js (hunk 2 above)
Comment (taniaprocha) [reply src/sections/CareersHero/index.js:6]: Yes!

**Pull request merged**